### PR TITLE
Add pending strings

### PR DESCRIPTION
**Pull request by Vinnl**
These are all strings I found in the Relay website repository that hadn't been submitted yet.

Note that @codemist will probably also add a few more from https://github.com/mozilla/fx-private-relay/pull/2552, so we'll probably want to wait on that before merging this.

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -36,6 +36,12 @@ nav-profile-contact = Contact us
 # This is only visible to Premium users.
 nav-profile-contact-tooltip = Get in touch about { -brand-name-relay-premium }
 nav-profile-image-alt = { -brand-name-firefox-account(capitalization: "uppercase") } Avatar
+nav-dashboard = Dashboard
+nav-phone = Phone Number
+nav-duo-description = Switch dashboards
+nav-duo-email-mask-alt = Email masks
+nav-duo-phone-mask-alt = Phone masks
+
```
Comment (Vinnl) [inline en/app.ftl:44]: These strings are for the switcher between the email and phone dashboards on mobile:

image

File: en/phones.ftl [hunk 2]
```diff
@@ -133,3 +133,66 @@ phone-banner-cta-user = Upgrade now
 #   $nr_texts (number) - the number of texts one can send and receive per month
 phone-banner-float-limits = { $nr_calls } call minutes + { $nr_texts } texts
 phone-banner-float-replies = Reply to texts privately
+
+## Frequently Asked Questions about Phone plans
```
Comment (Vinnl) [inline en/phones.ftl:137]: These can all be seen at https://relay.firefox.com/faq/ if you're in the US or Canada.

File: en/app.ftl [hunk 3]
```diff
@@ -492,6 +498,7 @@ settings-api-key-description-bolded = Only use this with apps that you trust to
 # This is a label that appears on hover to copy the API key
 settings-button-copy = Click to copy
 setting-api-key-copied = Copied!
+setting-api-key-copied-alt = Click to copy
```
Comment (Vinnl) [inline en/app.ftl:501]: This is on the settings page, the text for screen readers:

image

**Comment by codemist** [comment]
@Vinnl I just added a couple from https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/106

File: en/phones.ftl [hunk 4]
```diff
@@ -133,3 +133,66 @@ phone-banner-cta-user = Upgrade now
...
+phone-masking-faq-answer-can-i-block = You can block all forwarding from a single number.
+
+phone-masking-faq-question-spam = What if my phone mask starts getting spam?
+phone-masking-faq-answer-spam = If you start getting spam, you can block the numbers sending you spam.
+
+phone-masking-faq-question-disable-logging = Can I disable the logging of callers or text senders?
+phone-masking-faq-answer-disable-logging = Yes, you can disable logging of numbers from the { -brand-name-relay } dashboard. However, you will no longer be able to reply to texts or block specific numbers, because the log is how we are able to track who sent you a text message.
+
+phone-masking-faq-question-can-i-share = Can I share the number that forwards me text messages?
+phone-masking-faq-answer-can-i-share = If you share this number, nothing will happen—this number is not your phone number mask. It is just the contact number from which { -brand-name-relay } will forward your texts and calls.
```
Comment (peiying2) [inline en/phones.ftl:188]: Add spaces before and after the dash.

File: en/phones.ftl [hunk 5]
```diff
@@ -133,3 +133,66 @@ phone-banner-cta-user = Upgrade now
...
+
+phone-masking-faq-question-how-i-save-card = How do I save the { -brand-name-relay } contact card?
+phone-masking-faq-answer-how-i-save-card = Once you upgrade to { -brand-name-relay } phone number masking, we will text you a contact card that contains the number from which you will receive forwarded calls and texts, similar to any contact card that stores the phone number of people who contact you. On most devices, you can select that contact card and save it like any other contact on your phone.
+
+phone-masking-faq-question-install-app = Do I need to install an app to use { -brand-name-relay } phone masking?
+phone-masking-faq-answer-install-app = No, { -brand-name-relay } phone masking works using your device’s standard text messaging and calling apps.
+#   $url (url) - link to Firefox Relay's Privacy Policy, i.e. https://www.mozilla.org/privacy/firefox-relay/
```
Comment (peiying2) [inline en/phones.ftl:195]: Should these two comments go above the `answer` string, so the localizers can see it. Right now, they are viewable along with the `question` string.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Overall looks good, with a couple of minor comments.

File: en/phones.ftl (hunk 4 above)
Comment (Vinnl) [reply en/phones.ftl]: 7edc42d

File: en/phones.ftl (hunk 5 above)
Comment (Vinnl) [reply en/phones.ftl:196]: Ah yes: aee42ad

**Review by peiying2 — APPROVED** [review]
Thanks for the updates. LGTM.

**Pull request merged**